### PR TITLE
migration: Update to UnreleasedTreasury contract

### PR DESCRIPTION
**Pull request by palango**
Update migration for the renaming of the `CeloUnreleasedTreasury` contract.

### CODE REVIEW

File: op-chain-ops/cmd/celo-migrate/state.go [hunk 1]
```diff
@@ -80,8 +80,8 @@ var (
 			common.HexToAddress("0xba5Ed099633D3B313e4D5F7bdc1305d3c28ba5Ed"): false,
 		},
 	}
-	distributionScheduleAddressMap = map[uint64]common.Address{
-		AlfajoresNetworkID: common.HexToAddress("0x78af211ad79bce6bf636640ce8c2c2b29e02365a"),
+	unreleasedTreasuryAddressMap = map[uint64]common.Address{
+		AlfajoresNetworkID: common.HexToAddress("0x07bf0b2461A0cb608D5CF9a82ba97dAbA850F79F"),
```
Comment (karlb) [inline op-chain-ops/cmd/celo-migrate/state.go:84]: So it's not only renamed but also has a new address. I assume this is intentional and correct.

File: op-chain-ops/cmd/celo-migrate/state.go [hunk 2]
```diff
@@ -139,7 +139,7 @@ func applyStateMigrationChanges(config *genesis.DeployConfig, l2Allocs types.Gen
 	// Initialize the distribution schedule contract
```
Comment (karlb) [inline op-chain-ops/cmd/celo-migrate/state.go]:
```suggestion
	// Initialize the unreleased treasury contract
```

File: op-chain-ops/cmd/celo-migrate/state.go [hunk 3]
```diff
@@ -139,7 +139,7 @@ func applyStateMigrationChanges(config *genesis.DeployConfig, l2Allocs types.Gen
...
 	// This uses the original config which won't enable recent hardforks (and things like the PUSH0 opcode)
 	// This is fine, as the token uses solc 0.5.x and therefore compatible bytecode
-	err = setupDistributionSchedule(db, cfg)
+	err = setupUnreleasedTreasury(db, cfg)
 	if err != nil {
 		// An error here shouldn't stop the migration, just log it
 		log.Warn("Error setting up distribution schedule", "error", err)
```
Comment (karlb) [inline op-chain-ops/cmd/celo-migrate/state.go]:
```suggestion
		log.Warn("Error setting up unreleased treasury", "error", err)
```

**Review by karlb — COMMENTED** [review]
Just two missing renames.

**Review by karlb — APPROVED** [review]

File: op-chain-ops/cmd/celo-migrate/state.go (hunk 1 above)
Comment (palango) [reply op-chain-ops/cmd/celo-migrate/state.go:84]: Yes, the scope of the contract changes and it was deployed again.

**Pull request merged**